app/src/views/home/maximize_screen.tsx

```diff
@@ -137,7 +137,7 @@ export default function MaximizeScreen({
           <ThemedView style={styles.imageContainer}>
             <Image
               testID="post-image"
-              source={postImage ? { uri: postImage } : require(noImage)}
+              source={postImage ? { uri: postImage } : noImage}
```

Comment:
would it not be more efficient to compute the require and the path in the if so that you don't compute it even if not used?

Reply:
You're right thanks !